test/data/deploy/inference-service-without-model-version.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  name: dummy-inference-service
+  namespace: default
+  labels:
+    "mr-registered-model-id": "1"
```

Comment:
if the model version is not added, it means that the version is `1` ?

Reply:
if the modelVersion is not added it means the latest version in that moment